src/shared/api.ts

```diff
@@ -2509,9 +2529,9 @@ export const cerebrasModels = {
 		outputPrice: 0,
 		description: "SOTA coding performance with ~2500 tokens/s",
 	},
-	"qwen-3-235b-a22b": {
```

Comment:
Reason for removal? & what would happen to people who are using this model currently?

Reply:
<img width="902" height="316" alt="image" src="https://github.com/user-attachments/assets/3568cd34-8fec-4d41-84c2-b38bc108ef95" />